test-common/src/main/java/com/gentics/mesh/test/docker/NoWaitStrategy.java

```diff
@@ -1,6 +1,6 @@
 package com.gentics.mesh.test.docker;
 
-import org.testcontainers.containers.GenericContainer.AbstractWaitStrategy;
+import org.testcontainers.containers.wait.strategy.AbstractWaitStrategy;
```

Comment:
Why was this changed?

Reply:
I confirm this - newer versions of testcontainers have modified API.